src/App.vue

```diff
@@ -49,11 +50,114 @@ export default {
       // Message type (Info, Success, or Error) to display on banner
       messageType: 'Info',
       // API key from openweathermap.org - Unique to each person
-      openweathermapApiKey: '',
-      // Batul: ""
+      openweathermapApiKey: '5b59d0f6dbbe87dd97035b964b2b163b',
```

Comment:
rather storing raw keys in vue files can we introduce .env or config and from there we should be using private configs here?